### DISCUSSION TITLE
[BUG] far selection does not work

### DISCUSSION THREAD

**Discussion by nikoloz110** [rxhanson/Charmstone-Community#30] [Q&A]
title basically.  if "far selection distance" is set to other than 0, when moving cursor far, the application icon appears, but it does not get launched, or highlighted.

thanks, and feel free to reach out for logs or other info might help you to debug. 

charmstone: 0.29
mac os: 13.3.1 (a) 
macbook pro 15-inch, 2017 (intel based)

**Comment by nikoloz110** [comment]
actually I had to increase outer safe area to at least 50% of the bar, to make it work.  can be quite tricky to figure out.  the issue is closed on my end, but will leave it open in case you wonna make a comment, if not you can close it ;)

**Comment by rxhanson** [comment]
Glad you figured out a resolution here, but something definitely doesn't sound right. It could be just that Charmstone works in a way that was not your expectation. If you come across a scenario where you want me to weigh in on intended functionality, feel free to make a screen recording so that I might be able to better understand the expectation.
